vars/kolaTestIso.groovy

```diff
@@ -1,102 +1,36 @@
 // Run kola testiso
 // Available parameters:
-//     arch:               string  -- the target architecture
 //     cosaDir:            string  -- cosa working directory
 //     extraArgs:          string  -- extra arguments to pass to `kola testiso`
-//     extraArgs4k:        string  -- extra arguments to pass to 4k `kola testiso`
-//     extraArgsMultipath: string  -- extra arguments to pass to multipath `kola testiso`
-//     extraArgsUEFI:      string  -- extra arguments to pass to UEFI `kola testiso`
-//     scenarios:          string  -- scenarios to pass to `kola testiso`
-//     scenarios4k:        string  -- scenarios to pass to `kola testiso`
-//     scenariosMultipath: string  -- scenarios to pass to `kola testiso`
-//     scenariosUEFI:      string  -- scenarios to pass to `kola testiso`
-//     skipMetal4k:        boolean -- skip metal4k image
-//     skipMultipath:      boolean -- skip multipath tests
-//     skipUEFI:           boolean -- skip UEFI tests
-//     skipSecureBoot      boolean -- skip secureboot tests
 //     marker:             string  -- some identifying text to add to uploaded artifact filenames
+//     skipSecureBoot      boolean -- skip secureboot tests
+
 def call(params = [:]) {
-    def arch = params.get('arch', "x86_64");
-    def cosaDir = utils.getCosaDir(params)
+    def cosaDir = utils.getCosaDir(params);
     def extraArgs = params.get('extraArgs', "");
-    def extraArgs4k = params.get('extraArgs4k', "");
-    def extraArgsMultipath = params.get('extraArgsMultipath', "");
-    def extraArgsUEFI = params.get('extraArgsUEFI', "");
-    def scenarios = params.get('scenarios', "");
-    def scenarios4k = params.get('scenarios4k', "");
-    // only one test by default
-    def scenariosMultipath = params.get('scenariosMultipath', "iso-offline-install");
-    // by default, only test that we can boot successfully
-    def scenariosUEFI = params.get('scenariosUEFI', "iso-live-login,iso-as-disk");
     def marker = params.get('marker', "");
+    def denyList = "--denylist-test ";
 
+    if (params['skipSecureBoot']) {
+        denyList += "*.uefi-secure"
+    } else if (params['skipUEFI']) {
```

Comment:
I think we need to flip this around so that we check `skipUEFI` first and then `skipSecureBoot` second. Or alternatively, don't make it an `else if`, but just a separate if statement.